eZ/Publish/API/Repository/Tests/FieldType/DateIntegrationTest.php

```diff
@@ -292,7 +292,7 @@ public function provideToHashData()
             [
                 DateValue::fromTimestamp($timestamp),
                 [
-                    'timestamp' => $dateTime->setTime(0, 0, 0)->getTimestamp(),
+                    'timestamp' => $dateTime->getTimestamp(),
```

Comment:
silly question: doesn't this work by accident because you haven't tested it at midnight? ;) 
I mean, I would expect `$dateTime` to have current time as well in this case, if not set.
But commenting based on this line only, haven't seen full context, so feel free to correct me if I'm wrong.

Reply:
@alongosz it doesn't matter what time I run the test at, because `$dateTime` is set with the predefined timestamp (186401) a few lines before. I'm not sure though if that answers your question :)